index.html

```diff
@@ -100,19 +100,26 @@ <h3>Project 6</h3>
     </section>
 
 
-    <!--
+    
 		<section class="contact">
-							<div id="contactme" class="container">
-								<h1>{Contact Me}</h1>
-								<p>If you like what you see, please feel free to get in touch to discuss your project requirements</p>
-							</div>
-						</section>	
-																		
-						<section class="container">
-							<footer>
-								<p>Matt Martin &copy2020</p>
-							</footer>
-						</section>-->
+		    <div id="contactme" class="container">
+		        <h1>{Contact Me}</h1>
+		        <p>If you like what you see, please feel free to get in touch to discuss your project requirements</p>
+				<h2>You can e-mail me at  <span>matt@webdev.com</span></h2>
+		    </div>
+		</section>
+
+		
+		
+		
+		<section class="container">
+			<div id="footer" class="container">
+		    	<footer>
+		        	<p>Matt Martin &copy2020</p>
+		    	</footer>
+		</div>
```

Comment:
This div closing tag needs indenting inline with it's opening tag. Maybe it could be removed completely and just the footer used instead.